File: src/main/java/com/addressbook/model/FieldDescription.java
package com.addressbook.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@AllArgsConstructor
@ToString
public class FieldDescription {

    private String name;
    private String displayName;
    private String width;
    private String type;

}

File: addressbook-main/src/main/kotlin/com/addressbook/model/FieldDescription.kt
package com.addressbook.model

class FieldDescription(name: String, displayName: String, width: String, type: String) {

    var name: String? = null
    var displayName: String? = null
    var width: String? = null
    var type: String? = null

    init {
        this.name = name
        this.displayName = displayName
        this.width = width
        this.type = type
    }
}
